documentation/book/con-configure-operator.adoc

```diff
@@ -0,0 +1,78 @@
+[id='configure-cluster-operator_{context}']
+= Configuring Cluster Operator
+
```

Comment:
why is this a concept? the content seems to be "dancing around" a procedure, which is actually missing.